packages/teams-js/test/public/settings.spec.ts

```diff
@@ -120,10 +191,10 @@ describe('settings', () => {
 
       expect(handlerCalled).toBe(true);
     });
-  });
 
-  it('should successfully register a remove handler', () => {
-    return utils.initializeWithContext('settings').then(() => {
+    it('settings.registerOnSaveHandler should successfully register a remove handler', async () => {
```

Comment:
I know you didn't originally write this test, but what exactly is a "remove handler"?

Reply:
It was issue with naming convention I have updated it.